cmd/node/config/economics.toml

```diff
@@ -27,16 +27,16 @@
 [FeeSettings]
     MaxGasLimitPerBlock     = "1500000000"
     MaxGasLimitPerMetaBlock = "15000000000"
-    MinGasPrice = "500000000" #will yield min tx fee of 0.01ERD
+    MinGasPrice = "100000000" #will yield min tx fee of 0.00005ERD
```

Comment:
Double checked, the fee is actually .000005 instead of .00005 What is the desired value?

Reply:
done